src/Testing/Contracts/SetUpFinderActionContract.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaraStrict\Testing\Contracts;
+
+use Symfony\Component\Finder\Finder;
+
+interface SetUpFinderActionContract
+{
+    public function execute(Finder $finder): Finder;
```

Comment:
Říkal jsem si jestli udělat factory na Finder, nebo jen takto mít možnost upravit?

Reply:
Dal bych factory jak píšeš a consumer si to pak přes container můžeš "switchnout"